packages/components/src/styles/list/list.stories.ts

```diff
@@ -29,370 +23,183 @@ export default {
       url: 'https://www.figma.com/file/DDSyYvf2q99RhiyDjy03s5/List?type=design&node-id=971-4578&mode=design&t=2UZo6NW6ErMA2G5X-0'
     }
   },
-  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
-  argTypes
-};
-
-export const Default = {
   args: overrideArgs({
     type: 'slot',
     name: 'default',
-    value: html`<li>
-      Lorem Ipsum
-      <ul>
-        <li>
-          Dolor sit
-          <ul>
-            <li>Amet</li>
-            <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-          </ul>
-        </li>
-      </ul>
-    </li>`
+    value:
+      '<ul class="sd-list"><li>Unordered list level 1<ul><li>Unordered list level 2<ul><li>Unordered list level 3</li></ul></li></ul></li><li>Unordered list level 1</li><li>Unordered list level 1</li></ul>'
```

Comment:
Please format so that it's easier to reader

Reply:
Updated